protos/sigstore_common.proto

```diff
@@ -79,6 +79,13 @@ enum PublicKeyDetails {
         // Ed 25519
         PKIX_ED25519 = 7; // See RFC8032
         PKIX_ED25519_PH = 8;
+
+        // **EXPERIMENTAL**: LMS and LM-OTS variants.
+        EXPERIMENTAL_LMS_SHA256 = 14;
+        EXPERIMENTAL_LMOTS_SHA256 = 15;
+
+        // Reserved for future additions of public key/signature algorithm types.
+        reserved 16 to 50;
```

Comment:
NB: I've added a reserved range here, similar to some of the other messages we have. I figure this will make things a little more flexible on the wire format side, although I'm not sure if it matters in the JSON serialization case.